src/js/filter.ts

```diff
@@ -109,14 +110,9 @@ const setUpAllFilters = (
   searchElement: Choices,
   sliders: PopulationSliders
 ): void => {
-  [
-    ".scope",
-    ".policy-change",
-    ".land-use",
-    ".implementation-stage",
-    "#no-requirements-toggle",
-  ].forEach((selector) => {
-    document.querySelector(selector).addEventListener("change", () => {
+  // "input" covers all checkboxes and sliders
```

Comment:
Do we want it to cover the sliders? I thought that needs to be set up separately? Should this be `input[type=checkbox]` instead?